templates/blog/post_detail.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>{{ post.title }}</title>
+{% endblock %}
+
+{% block content %}
+  <h2>{{ post.title }}</h2>
+  <div>
+    <div> Author: {{ post.owner }} Created: {{ post.created_time }}</div>
```

Comment:
Consider using Django's date filter for formatting the creation time. It can make the date more readable. You can use it like this: `{{ post.created_time|date:"F j, Y" }}`.